### PR TITLE
Prettying up the stack status

### PR DESCRIPTION
**Pull request by kiiadi**
screen shot 2018-11-14 at 12 31 23 pm

### CODE REVIEW

File: core/src/software/aws/toolkits/core/utils/StringUtils.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.core.utils
+
+/**
+ * Designed to convert underscore separated words (e.g. UPDATE_COMPLETE) into title cased human readable text
+ * (e.g. Update Complete)
+ */
+fun String.toHumanReadable() = split('_').map { it.toLowerCase().capitalize() }.joinToString(" ")
```
Comment (abrooksv) [inline core/src/software/aws/toolkits/core/utils/StringUtils.kt]: `=StringUtil.toTitleCase(this.lowercase().replace('_', ' '))` doesn't require creating arrays

Comment (kiiadi) [reply core/src/software/aws/toolkits/core/utils/StringUtils.kt]: Roger-that

File: core/src/software/aws/toolkits/core/utils/StringUtils.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (zhangzhx) [inline core/src/software/aws/toolkits/core/utils/StringUtils.kt]: Nit: dup license here

**Comment by kiiadi** [comment]
Fixed - moved to `TextUtils` so taht we can get at the JetBrains function

**Pull request merged**